drivers/selenium.rst

```diff
@@ -5,6 +5,13 @@ SeleniumDriver provides a bridge for the famous `Selenium`_ tool. If you
 need legacy Selenium, you can use it right out of the box in your Mink test
 suites.
 
+.. caution::
+
+    The SeleniumRC protocol used by this driver is deprecated and does not
+    support all Mink features. For this reason, the SeleniumDriver is deprecated
+    in favor of the :doc:`/drivers/selenium2`, which is based on the new
```

Comment:
Well, we have 2 choices: either absolute references (which need the leading slash) or relative references (which would be ``:doc:`selenium2` ``). I'm using absolute references everywhere currently (easier when moving a file to a different location as all references inside it don't need to be changed, and searching for references to a given file is also easier)